paper/main.tex

```diff
@@ -115,6 +116,13 @@ \section{Background}
     \item Do not be too clever by scheming to exploit the opponent
 \end{itemize}
 
+Forgiveness in R. Axelrod's terms is a two round procedure. Though essentially
```

Comment:
Maybe leave off R. Axelrod here unless you've verified that his works describe forgiveness as such.

Though ... --> Forgivess is a strategy's ...